src/dex-ui-components/base/Breadcrumb/__tests__/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render, screen, fireEvent, cleanup } from "@testing-library/react";
+import { Breadcrumb } from "../Breadcrumb";
+
+afterEach(() => {
```

Comment:
cleanup() may already be automatically called for each test:

> Please note that this is done automatically if the testing framework you're using supports the afterEach global and it is injected to your testing environment (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups after each test.

https://testing-library.com/docs/react-testing-library/api/#cleanup

I would double-check to see if the global cleanup is set up in our environment so we can remove the individual `afterEach(() => { cleanup(); });` calls from our test files.

Reply:
Global cleanup works when threads is set to "true" in vitest config. As default value of threads is anyway true, so removed the cleanup from test files.